### ISSUE TITLE
Document the workflow from swagger.yaml to go files

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/keptn#1331]
Right now there is a manual step involved in generating swagger yaml files, see e.g., https://github.com/keptn/keptn/tree/master/mongodb-datastore

```console
swagger generate server -A mongodb-datastore -f ./swagger.yaml --with-expand
```

1. It is unclear which version of `swagger` needs to be used,
2. It is unclear that this command also generates `.go` files
3. For devs it is not clear that this command needs to be executed everytime a change has been introduced, this should be automated!

**Comment by ermin-muratovic** [comment]
When executing `swagger generate server -A mongodb-datastore -f ./swagger.yaml --with-expand` the model configuration for request/response description is not correctly used and therefore some files are generated incorrect.

Dropping the ` --with-expand` fixed the issue of incorrect generated files.

**Comment by christian-kreuzberger-dtx** [comment]
Has been fixed with 0.6.1 release.
